app/controllers/state_file/questions/questions_controller.rb

```diff
@@ -30,6 +30,16 @@ def redirect_if_no_intake
         end
       end
 
+      def redirect_if_df_data_required
+        return if current_state_code == "nj"
```

Comment:
hm, i get that we're excluding NJ but if so, I highly recommend asking NJ to fix this on their end, or maybe pairing with them to get this fixed together?